core/chains/evm/gas/block_history_estimator.go

```diff
@@ -301,9 +296,6 @@ func (b *BlockHistoryEstimator) runLoop() {
 
 // FetchBlocksAndRecalculate fetches block history leading up to head and recalculates gas price.
 func (b *BlockHistoryEstimator) FetchBlocksAndRecalculate(ctx context.Context, head *evmtypes.Head) {
-	ctx, cancel := context.WithTimeout(ctx, maxEthNodeRequestTime)
```

Comment:
This was redundant because all queries already have a timeout automatically applied